src/_includes/footer.njk

```diff
@@ -1,3 +1,21 @@
-<footer>
-
+<footer class="bg-darkBlue">
+    <div class="text-white container py-5 md:py-10">
+        <div class="grid grid-cols-2 gap-x-20">
+            <div>
+                <a href="https://www.codeforamerica.org" title="Link to Code for America" target="_blank"><img src="/code-for-america-logo.svg" alt="Code for America logo" /></a>
+                <p class="mt-5">Open SGF is an official Brigade of <a class="text-blue underline" href="https://www.codeforamerica.org" title="Link to Code for America" target="_blank">Code for America</a>.</p>
+            </div>
+            <div class="grid grid-rows-2 place-self-end">
+                <ul class="flex items-center space-x-10 place-self-center pb-5 border-b-2">
+                    <li><a href="https://github.com/Open-SGF" title="Link to Open SGF Github" target="_blank"><img src="/github-logo.svg" alt="Github logo" /></a></li>
+                    <li><a href="https://discord.gg/jFD8dZP" title="Link to Open SGF Discord server" target="_blank"><img src="/discord-logo.svg" alt="Discord logo" /></a></li>
+                    <li><a href="https://www.meetup.com/open-sgf" title="Link to Open SGF Meetup" target="_blank"><img src="/meetup-logo.svg" alt="Meetup logo" /></a></li>
+                </ul>
+                <div class="place-self-center">
```

Comment:
@glitchedmob there's also one other problem with the footer, the `place-self-center` isn't super great because it wants to align elements dead center in a grid cell, and I couldn't find a way to make them align center and then vertically at the top of the cell. Should I forget about grid and just use flex?

Reply:
cf11c25 ("Justify social right and center vertically")